src/nih_wayfinding/app/scripts/views/locations/add-location/location-another/location-another-controller.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function LocationsAnotherController($state, NavbarConfig, UserLocations, ProfileService) {
+        var ctl = this;
+        initialize();
+
+        function initialize() {
+            NavbarConfig.set({ title: 'Custom Locations' });
+            ctl.optionClicked = optionClicked;
+            ctl.gridOptions = [
+                { text: 'Yes' },
+                { text: 'No thanks' }
+            ];
+        }
+
+      function optionClicked(option) {
+          goNext(option.text);
+      }
+
+      function goNext(answer) {
+          console.log(answer);
+          if (answer === 'yes') {
```

Comment:
This is fairly brittle, and I don't think it works as is, since 'Yes' !== 'yes'.

Consider adding an additional key to each gridOptions object, something like 'toState', where the value is the state id. Then, this can just be directly passed to $state.go().